ouroboros-network-framework/src/Ouroboros/Network/Server/RateLimiting.hs

```diff
@@ -41,6 +44,28 @@ data AcceptedConnectionsLimit = AcceptedConnectionsLimit {
   }
   deriving (Eq, Ord, Show)
 
+instance ToJSON AcceptedConnectionsLimit where
+  toJSON AcceptedConnectionsLimit
+          { acceptedConnectionsHardLimit
+          , acceptedConnectionsSoftLimit
+          , acceptedConnectionsDelay
+          } =
+    object [ "AcceptedConnectionsLimit" .=
```

Comment:
I'd avoid the top level wrapper, e.g. instead of
```json
{ "AcceptedConnectionsLimit": { "hardLimit": 500, "softLimit": 30, "delay": 60} }
```
I think it's better to just have:
```json
{ "hardLimit": 500, "softLimit": 300, "delay": 60}
```

Reply:
I would avoid breaking anything.

Reply:
Let's keep it like this then. @coot> Good for you?